src/main/scala/shine/DPIA/InferAccessAnnotation.scala

```diff
@@ -16,17 +16,23 @@ import scala.collection.mutable
 import scala.util.{Failure, Success, Try}
 
 object inferAccess {
-  def apply(e: r.Expr): MutableIdentityHashMap[r.Expr, PhraseType] =
+  def apply(e: r.Expr): java.util.IdentityHashMap[r.Expr, PhraseType] =
+  //MutableIdentityHashMap[r.Expr, PhraseType] =
```

Comment:
should we keep that?

Reply:
Nope. I will clean this up as well. Thanks for spotting this.